Userland/Libraries/LibWeb/Bindings/LocationObject.cpp

```diff
@@ -23,13 +23,13 @@ void LocationObject::initialize(JS::GlobalObject& global_object)
 {
     Object::initialize(global_object);
     u8 attr = JS::Attribute::Writable | JS::Attribute::Enumerable;
-    define_native_property("href", href_getter, href_setter, attr);
-    define_native_property("host", host_getter, nullptr, attr);
-    define_native_property("hostname", hostname_getter, nullptr, attr);
-    define_native_property("pathname", pathname_getter, nullptr, attr);
-    define_native_property("hash", hash_getter, nullptr, attr);
-    define_native_property("search", search_getter, nullptr, attr);
-    define_native_property("protocol", protocol_getter, nullptr, attr);
+    define_native_accessor("href", href_getter, href_setter, attr);
+    define_native_accessor("host", host_getter, nullptr, attr);
+    define_native_accessor("hostname", hostname_getter, nullptr, attr);
+    define_native_accessor("pathname", pathname_getter, nullptr, attr);
+    define_native_accessor("hash", hash_getter, nullptr, attr);
+    define_native_accessor("search", search_getter, nullptr, attr);
+    define_native_accessor("protocol", protocol_getter, nullptr, attr);
```

Comment:
Didn't you want to replacethe `nullptr`s with `{}`? :)

Reply:
Oh yeah, totally forgot!